module-1/lab-functional-programming/your-code/Q1.ipynb

```diff
@@ -19,49 +19,105 @@
   },
```

Comment:
Great job Constant!! Nothing to comment. Remember to keep the code as simple as possible and as readable as possible. Take out all your personal notes from the final notebook if possible. In case you want to try more than one approach in each part of the exercise, comment it on your code. This way I will know that you are trying different ways for the same output. 
Note: Remember to use relative paths always as I commented in your bag of words notebook.